Cargo.toml

```diff
@@ -31,7 +31,7 @@ categories = ["development-tools", "wasm", "web-programming"]
 keywords = ["leptos"]
 version = "0.2.27"
 edition = "2021"
-rust-version = "1.71"
+rust-version = "1.82.0"
```

Comment:
Clippy was giving warnings that the earliest rust version to support [`is_none_or` (which is used in this project)](https://github.com/leptos-rs/cargo-leptos/blob/main/src/config/bin_package.rs#L90) is 1.82.0.